syntax = "proto3";

package wobl.msg; 

message JointState {
    double timestamp = 1;
    repeated float position = 2;
    repeated float velocity = 3;
    repeated float effort = 4;
}

message JointCommand {
    double timestamp = 1;
    repeated float position = 2;
    repeated float velocity = 3;
}

message JointEnable {
    double timestamp = 1;
    bool enable = 2;
}